docs/samples/modelmesh_sdk_sample.ipynb

```diff
@@ -0,0 +1,390 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample on how to use modelmesh-serving with SDK v1beta1 python\n",
+    "\n",
+    "## This notebook shows how to use the python sdk with ModelMesh to create an InferenceService"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kubernetes import client\n",
+    "from kserve import constants\n",
+    "from kserve import V1beta1InferenceService\n",
+    "from kserve import V1beta1InferenceServiceSpec\n",
+    "from kserve import V1beta1PredictorSpec\n",
+    "from kserve import V1beta1SKLearnSpec\n",
+    "from kserve import V1beta1TFServingSpec\n",
+    "from kserve import V1beta1StorageSpec\n",
+    "from kserve import KServeClient"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the global variables to be used in InferenceService creation"
```

Comment:
I don't think this should be a header and there could be a short explanation on the variables being set up.
- The `namespace` definition is where the InferenceService will be deployed to
- `name` will be the name of the InferenceService
- For ModelMesh, the `v2` protocol must be used since it doesn't support the default `v1` protocol